### PR TITLE
Added Asthma Detection ML Model

### LINKED ISSUE #200: 💡[Feature]:  Add asthma detection
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Feature Description

Can add asthma detection 
This project predicts whether the person is suffering from the respiratory disease of Asthma or not. The porject classifies the person into three main categories as suffering from mild asthma, suffering from Moderate or high asthma and Not suffering from asthma on streamlit based application.projectStreamlit

### Use Case

The project classifies the person into three main categories: suffering from mild asthma, Moderate or high asthma, and Not suffering from asthma on streamlit-based application.

The application takes input like the person has various disorders like tiredness, running nose,nasal congestion, difficulty in breathing or not, age group , gender, and predict the likeliness of the disease

### Benefits

The script will allow you to predict whether the person is suffering from asthma or not based on input features.

### Add ScreenShots

_No response_

### Priority

High

### Record

- [X] I have read the Contributing Guidelines
- [X] I'm a GSSOC'24 contributor
- [X] I want to work on this issue

### PR DESCRIPTION
**Pull request by debangi29**
Fixed #200 
Added ML model and python file for Asthma detection

### CODE REVIEW

**Comment by debangi29** [comment]
Please check thr PR @TAHIR0110 
Thanks :)

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]
@debangi29 Looks good. You have created a streamlit app. So, attach a working video of this project?

**Comment by debangi29** [comment]
I have added the working video file of the detection model

File: Asthma_Detection/Asthma_Prediction_App.mp4
Comment (Avdhesh-Varshney) [inline Asthma_Detection/Asthma_Prediction_App.mp4:1]: No, don't commit it in project. Delete it from here and show me in the conversation only.

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]

**Comment by debangi29** [comment]
Okay, I have deleted it... 
Heres the video :)

https://github.com/TAHIR0110/ThereForYou/assets/117537653/f954c2b5-070b-41d2-afbc-cbfbf118be80

**Review by Avdhesh-Varshney — APPROVED** [review]

**Pull request merged**